include/wil/result_macros.h

```diff
@@ -1689,6 +1689,19 @@ enum class ErrorReturn
     None
 };
 
+/// @cond
+namespace details
+{
+    // 'FARPROC' is declared in such a way that it cannot safely be assumed cast-able to other function pointer types.
+    // This function helps alleviate warnings that can arise from this
+    template <typename FuncPtr>
+    inline FuncPtr GetProcAddress(_In_ HMODULE module, _In_ LPCSTR procName) WI_NOEXCEPT
+    {
+        return reinterpret_cast<FuncPtr>(reinterpret_cast<void (*)()>(::GetProcAddress(module, procName)));

```

Comment:
I'm slightly concerned about the lack of calling convention here, however everything seems to be building and running fine for x86, so I'll probably avoid messing with it for now